releases/release-5.0.6.md

```diff
@@ -0,0 +1,169 @@
+    - Avoid false positive error log about `invalid cop task execution summaries length` when running MPP query. [#28262](https://github.com/pingcap/tidb/pull/28262)
+    - Fixed a bug that causes TiDB panic when inserting invalid date value concurrently. [#28198](https://github.com/pingcap/tidb/pull/28198)
+    - Fix `can not found column in Schema column` error for mpp queries [#28147](https://github.com/pingcap/tidb/pull/28147)
+    - Fix a bug that TiDB may crash when TiFlash is shutting down. [#28138](https://github.com/pingcap/tidb/pull/28138)
+    - None. [#27927](https://github.com/pingcap/tidb/pull/27927)
+
++ TiKV/TiKV
+
+    - Fix resolved ts lag increased after stoping a tikv [#11538](https://github.com/tikv/tikv/pull/11538)
+    - Fix connection abort when too many raft entries are batched into one messages [#11532](https://github.com/tikv/tikv/pull/11532)
+    - Fix panic in rare conditions when merge, conf change and snapshot happen at the same time [#11510](https://github.com/tikv/tikv/pull/11510)
+    - Fix the issue that reverse scan can't detect memory locks and may read stale data. [#11453](https://github.com/tikv/tikv/pull/11453)
+    - Fix GC scan effectiveness to avoid OOM [#11419](https://github.com/tikv/tikv/pull/11419)
+    - None. [#11381](https://github.com/tikv/tikv/pull/11381)
+    - fix negative sign when decimal divide to zero [#11333](https://github.com/tikv/tikv/pull/11333)
+    - Fix incorrect by-instance gRPC average duration. [#11327](https://github.com/tikv/tikv/pull/11327)
+    - move verify_checksum to import-thread from apply-thread. [#11257](https://github.com/tikv/tikv/pull/11257)
+    - Fix label leaking of thread metrics [#11201](https://github.com/tikv/tikv/pull/11201)
+    - Fix CDC panic due to missing downstream. [#11136](https://github.com/tikv/tikv/pull/11136)
+    - Fix frequent CDC incremental scan retry due to `Congest` error. [#11090](https://github.com/tikv/tikv/pull/11090)
+    - fix channel full could break the raft connection [#11070](https://github.com/tikv/tikv/pull/11070)
+    - Hide untouched storage commands' metrics in grafana dashboard [#11001](https://github.com/tikv/tikv/pull/11001)
+    - Do not delete import files during Lightning import. [#10740](https://github.com/tikv/tikv/pull/10740)
+    - copr: fix Max/Min bug when comparing signed and unsigned int64 [#10617](https://github.com/tikv/tikv/pull/10617)
+    - - fix follower meta corruption in rare cases with more than 4 replicas [#10500](https://github.com/tikv/tikv/pull/10500)
+    - ```release-note [#10361](https://github.com/tikv/tikv/pull/10361)
+    - copr cast invalid utf8 string to real bug fix [#9871](https://github.com/tikv/tikv/pull/9871)
+
++ PingCAP/TiFlash
+
+    - Fix potential data inconsistency when widen pk column type if pk is handle [#3572](https://github.com/pingcap/tics/pull/3572)
```

Comment:
replace it with:

- Fix potential data inconsistency when widen pk column type if pk is handle [#3572](https://github.com/pingcap/tics/pull/3572)
- Fix Tiflash arm build when include `libnsl.so`. Try copy `libnsl.so` to `INSTALL_DIR` [#3205](https://github.com/pingcap/tics/pull/3205)
- Fix store size on `tiflash metrics` is not consistent.Decrease the size of checkpoint file when removing it  [#3187](https://github.com/pingcap/tics/pull/3187)
- Fix exception: `GC` removed normal file which just created [#3226](https://github.com/pingcap/tics/pull/3226)
- Fix sync schema exception when user upgrade the TiFlash version. [#2701](https://github.com/pingcap/tics/pull/2701)
- Fix tiflash randomly crash when a mpp query is killed. [#3448](https://github.com/pingcap/tics/pull/3448)
- Fix throw not constants exception in `substringUTF8` [#3266](https://github.com/pingcap/tics/pull/3266)
- Fix decode DAG request failed.Add retry when decode DAG request failed. [#3678](https://github.com/pingcap/tics/pull/3678)
- Fix the bug that results of `where <string>` is wrong because it will be converted to int type. [#3477](https://github.com/pingcap/tics/pull/3477)
- Fix `main_capacity_quota_` check  [#3425](https://github.com/pingcap/tics/pull/3425)
- Fix the inconsistent behavior of `CastStringAsDecimal` between tiflash and tidb/tikv. [#3674](https://github.com/pingcap/tics/pull/3674)
- Fix exception: `expected Nullable(Int64), got Int64`. Sending schemas in exchangeSender to avoid schema mismatch. [#3369](https://github.com/pingcap/tics/pull/3369)
- Fix the issue of unexpected error that `Unexpected type of column: Nullable(Nothing)` [#3376](https://github.com/pingcap/tics/pull/3376)
- Fix the issue that comparison between Decimal may cause overflow and report Can't compare. [#3097](https://github.com/pingcap/tics/pull/3097)